testing/trino-testing-containers/src/main/java/io/trino/testing/containers/Alluxio.java

```diff
@@ -43,8 +47,16 @@ public Alluxio()
 
     public void start()
     {
-        alluxioMaster.start();
-        alluxioWorker.start();
+        Failsafe.with(RetryPolicy.<Void>builder()
+                        .withMaxAttempts(10)
+                        .withBackoff(100, 500, MILLIS)
+                        .build())
+                .run(alluxioMaster::start);
```

Comment:
Do we need to rebuild the container anew, or forcefully shutdown previous start attempt?